.github/workflows/test-package.yml

```diff
@@ -59,27 +59,3 @@ jobs:
       - name: Run VM tests
         run: dart test --platform vm
         if: always() && steps.install.outcome == 'success'
-
```

Comment:
Was this removed intentionally?

Reply:
Yes. This was running on an old SDK that is no longer supported. I'll add testing on `2.12.0` in a followup.